src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        data-qa="keypress"
+        class="big-search-bar"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <img
+        src="./images/Search.svg"
+        alt="Search Icon"
+        class="image "
+      >
+    </form>
+
+    <form
+      action="#"
+      class="form"
+      method="post"
+      data-qa="small"
+    > <input
+        type="search"
+        class="small-search-bar"
```

Comment:
```suggestion
        class="search-bar small-search-bar"
```
2 point in checklist